test/functional/FunctionalCodeAction.hs

```diff
@@ -547,6 +547,31 @@ missingPragmaTests = testGroup "missing pragma warning code actions" [
                     , "f Record{a, b} = a"
                     ]
 
+            liftIO $ T.lines contents @?= expected
+    , testCase "After Shebang" $ do
+        runSession hlsCommand fullCaps "test/testdata/addPragmas" $ do
+            doc <- openDoc "AfterShebang.hs" "haskell"
+
+            _ <- waitForDiagnosticsFrom doc
+            cas <- map fromAction <$> getAllCodeActions doc
+
+            liftIO $ "Add \"TypeApplications\"" `elem` map (^. L.title) cas @? "Contains TypeApplications code action"
```

Comment:
(For a separate PR: these `@?` messages for the `addPragmas` tests are super confusing, and it is printed when `cas` **does not** contain `TypeApplications` action.)

Reply:
Would definitely look into this once I finish up with this. Taking up a lot of time from you